sbt-scalajs-bundler/src/main/scala/scalajsbundler/Commands.scala

```diff
@@ -3,6 +3,10 @@ package scalajsbundler
 import sbt._
 
 object Commands {
+  private val npm = sys.props("os.name").toLowerCase match {
+    case os if os.contains("win") ⇒ "cmd /c npm"
```

Comment:
It's testing what the current operating system is. Then it uses the built in `cmd` executable in windows to execute the `npm` batch file.

I don't really know why but it seems as though executing a shell command doesn't pick up windows batch files, only executables.

So if it's windows it executes `cmd /c npm update` etc, if it's some other os, just `npm update`. (See the usage of the npm val in string interpolation below).

I got the sys.props trick by looking at the sbt-native-packager source code.

Not really certain if there is better approach but this seems to work.